### DISCUSSION TITLE
filter by fields in other table

### DISCUSSION THREAD

**Discussion by damiankomonski** [prisma/prisma#22330] [Q&A]
### Question
```js
const result = await prisma.user.findFirst({
  select: {
    posts: {
      where: {
        fieldA: "abc",
      },
      orderBy: {
        title: 'asc',
      },
      select: {
        title: true,
      },
    },
  },
})
```

hello, I use it, I want filter using fileds in other table but this doesnt work, I get error 
`Unknown arg `where` in select.user.where for type user. Did you mean `select`?`

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by damiankomonski** [comment]
Also why i get such error?
also doesnt work... 😮 

```js
        where: {
          a: {
            some: {
            ~~~~
              name: {
                in: [
                  'b',
                  'cos'
                ]
              }
            }
          }
        }
      })
```
Unknown arg `some` in where.a.some for type aRelationFilter. Did you mean `is`? Available args:

**Comment by ludralph** [comment]
Hi @damiankomonski 👋 

Thank you for raising this question.

Can you please check the documentation on how to [filter by relations](https://www.prisma.io/docs/orm/prisma-client/queries/relation-queries#relation-filters)? You may also provide the schema and the exact kind of filtering you want in words and I could help come up with the query for it.

**Reply by damiankomonski** [reply]
I get error which i wrote. I dont know why.

**Reply by ludralph** [reply]
Hi @damiankomonski , can you please share your schema to debug this further? I tried locally with this query
```js
  const res = await prisma.user.findFirst({
    select: {
      posts: {
        where: {
          title: 'Hello World',
        },
        orderBy: {
          title: 'asc',
        },
        select: {
          title: true,
        },
      },
    },
  })
```
it produced this result set
```
{ posts: [ { title: 'Hello World' } ] }
```
If you can share a minimal reproduction in a Github repository, I can attempt to provide a solution.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
